app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -265,6 +267,29 @@ a.inplace-editing--trigger-link,
     &.edit-strategy-textarea
       margin-top: auto
 
+// Explicit styles for comment / input-esque trigger appearance
+.work-packages--activity--add-comment
+  margin: 20px 0
+
+  .inplace-editing--container
+    border: 1px solid #eee
+
+  .inplace-edit--read-value
+    padding: 10px 5px
+
+  .inplace-editing--trigger-link
+    color: #888
+    font-size: 1rem
```

Comment:
With your reasoning, I think that at least description and comments should look identical.

I took the comment form because it is *outside* the usual field attribute groups, and thus one can quickly miss it. With the current styling it looks more similar to a regular input field.